setup.py

```diff
@@ -41,24 +41,21 @@
       description="Multi-dimensional scaling approximations",
       long_description="Benchmarking several multi-dimensional scaling "
                        "approximation methods",
-      author="mdsa development team",
-      author_email="antgonza@gmail.com",
-      maintainer="mdsa development team",
-      maintainer_email="antgonza@gmail.com",
+      author="Hannes Holste and Knight Lab",
```

Comment:
Should this only be Knight Lab?

Reply:
My plan is to put the full list of contributors in the README right before we freeze the repo for the paper submission, since we have to collect a list of everyone who contributed anyway at a later stage. To my understanding setup.py author metadata only matters for pip package publication (right?), and it seems unlikely we'll make this a standalone package.

Reply:
OK, then should we remove?

Reply:
sure